aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,73 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+## When to use
 
-Forms provide a way to collect important data from a user. However, this data is only collected effectively when Forms are built in a way in which users are inclined to fill out in full.
+To efficiently capture information necessary for completing a task.
 
-### Usage
+## When not to use
 
-Forms don't need to be a tedious task for users to complete, as they often come to be. It is important to ease as much of this effort as possible when building a Form. Making Forms simple to complete leads to increase in form completion.
+Complicated processes where the user might need guidance at various stages, or when the form would be intimidatingly long. In these cases use a [Wizard](https://design-system.hpe.design/templates/wizard?q=error#guidance).
```

Comment:
A Wizard is still build using a form, so I'm wondering how we might clarify that.

Reply:
Think of it from the user's standpoint: we are giving them an alternative to using ONE basic form. A wizard.